README.md

```diff
@@ -1118,11 +1114,15 @@ Implementations MAY use the `validityParameters` field to specify custom "soft c
 
 ### Schema: `QuoteStub`
 
-Defines a public "quote stub," indicating a price bound for a given maker/taker asset pair that a trader may request a full quote for at a later time (see `dealer_getQuoteFromStub`).
+Defines a public "quote stub," indicating a price bound and quantity limit for a given maker/taker asset pair that a trader may request a full quote for at a later time (see `dealer_getQuoteFromStub`).
+
+@ todo: reconsider this, perhaps create + cancel instead
 
 Stubs are created and updated in a drop-and-replace manner, meaning a new updated stub should reference the one it is replacing, as a convenience to traders.
 
-At least one of either `makerSizeRequest` or `takerSizeRequest` and at least one of either `takerSizeBand` or `makerSizeBand` MUST be included in each stub.
+`makerSizeLimit` and `takerPriceBand` MUST be included in each stub.
+
+Some dealer's may service quote requests denominated in a quantity of either the maker or taker asset (@todo: how to tell?). If this is the case, a taker can infer the minimum available quantity of taker asset by using the `makerSizeLimit` and the right price bound defined in `takerPriceBand`.
 
 If both `makerSizeRequest` and `takerSizeRequest` (and their corresponding size bands) are included, that indicates the implementation provides quotes where the trader can denominate the size in either the asset they are sending (the taker asset) or the asset they are receiving (the maker asset). The dealer implementation would fill in the other value, similar to how regular [quotes](#quotes) work.
```

Comment:
This section needs to be amended or removed it `makerSizeRequest` and `takerSizeRequest` are no longer included.